https-did/build.gradle

```diff
@@ -0,0 +1,62 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
```

Comment:
I suppose this module should also be usable independent of the whole SDK so please add an: `apply plugin: "maven"` here